benchmarks/CMakeLists.txt

```
@@ -84,8 +83,11 @@ ConfigureBench(DYNAMIC_MAP_BENCH "${DYNAMIC_MAP_BENCH_SRC}")
 
 ###################################################################################################
 # - static_map benchmarks -------------------------------------------------------------------------
-set(STATIC_MAP_BENCH_SRC "${CMAKE_CURRENT_SOURCE_DIR}/hash_table/static_map_bench.cu")
-ConfigureBench(STATIC_MAP_BENCH "${STATIC_MAP_BENCH_SRC}")
+ConfigureNVBench(STATIC_MAP_BENCH
+  hash_table/static_map/contains_bench.cu
+  hash_table/static_map/erase_bench.cu
+  hash_table/static_map/find_bench.cu
+  hash_table/static_map/insert_bench.cu)
```

```suggestion
  hash_table/static_map/insert_bench.cu
  hash_table/static_map/find_bench.cu
  hash_table/static_map/contains_bench.cu
  hash_table/static_map/erase_bench.cu)
```

Most times, we want to take a look at the `insert` and `find` perf, so we should move those to the top.

---

Done. Eventually, this will be formatted by CMake format following alphabetical order.